tests/Integration/RequestManagementSystem/Test_Client_Req.py

```diff
@@ -78,32 +79,50 @@ def tearDown(self):
 
 
 class ReqClientMix(ReqClientTestCase):
+  def _checkSummary(self, initial, changes):
```

Comment:
Do I understand correctly that this checks whether the DB has changed accordingly to some expected changes ? You may want to add some comments :-D